lib/yard-api/templates/helpers/html_helper.rb

```diff
@@ -13,12 +13,19 @@ def url_for_file(file, anchor = nil)
 
   def static_pages()
     @@static_pages ||= begin
```

Comment:
Maybe this change warrants that we rename this to `@@visible_static_pages` for clarity and then rename `@@all_static_pages` to just `@@static_pages`.

What do you think?